src/components/code-embed/resizeable-panels.module.scss

```
@@ -82,6 +82,7 @@
 		width: var(--resizeable-panel_icon_height);
 		border-radius: var(--spc-1x);
 		background-color: var(--resizeable-panel_separator_background);
+		// TODO(undefined-css-vars): found --resizeable-panel_separator_border_width_focused, code expects --resizeable-panel_separator_outline_width_focused
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)